From 162324e24b3c5a6dcccd7faf2b0d65e6ccba1997 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Wed, 6 Sep 2023 16:46:49 +0200
Subject: [PATCH 0147/1695] dt-bindings: stm32: fix CLKSRC for RTC in stm32mp13
 clock bindings

Bad copy/paste, use MUX ID to configure the clock source of RTC and not
the clock ID.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git cd1a0e6f7c5e55e9a6cb4222f54604e7b4c66177 dt-bindings: stm32: fix CLKSRC for RTC in stm32mp13 clock bindings]
Change-Id: I9613d6cae8a2a843895713706ba615b4673bc707
Fixes: 19a4632e0f179 ("dt-bindings: stm32: add stm32mp13 clock and reset bindings")
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
---
 core/include/dt-bindings/clock/stm32mp13-clksrc.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/core/include/dt-bindings/clock/stm32mp13-clksrc.h b/core/include/dt-bindings/clock/stm32mp13-clksrc.h
index 209ad5d58..f1f0d8775 100644
--- a/core/include/dt-bindings/clock/stm32mp13-clksrc.h
+++ b/core/include/dt-bindings/clock/stm32mp13-clksrc.h
@@ -156,10 +156,10 @@
 #define CLK_PLL4_HSE		CLKSRC(MUX_PLL4, 1)
 #define CLK_PLL4_CSI		CLKSRC(MUX_PLL4, 2)
 
-#define CLK_RTC_DISABLED	CLKSRC(RTC, 0)
-#define CLK_RTC_LSE		CLKSRC(RTC, 1)
-#define CLK_RTC_LSI		CLKSRC(RTC, 2)
-#define CLK_RTC_HSE		CLKSRC(RTC, 3)
+#define CLK_RTC_DISABLED	CLKSRC(MUX_RTC, 0)
+#define CLK_RTC_LSE		CLKSRC(MUX_RTC, 1)
+#define CLK_RTC_LSI		CLKSRC(MUX_RTC, 2)
+#define CLK_RTC_HSE		CLKSRC(MUX_RTC, 3)
 
 #define CLK_MCO1_HSI		CLK_SRC(CK_MCO1, 0)
 #define CLK_MCO1_HSE		CLK_SRC(CK_MCO1, 1)
-- 
2.47.2

